test/lib/plugins/labels.test.js

```diff
@@ -76,7 +76,7 @@ describe('Labels', () => {
           repo: 'test',
           oldname: 'new-description',
           name: 'new-description',
-          color: '000000',
+          color: '#000000',
```

Comment:
since this is the assertion of what is sent in the api call, we would want the `#` to be removed by this point. i think this will also get the failing build back to passing

```suggestion
          color: '000000',
```